tests/unit/lib/test_datachain.py

```diff
@@ -9,7 +9,6 @@
 import pyarrow.parquet as pq
 import pytest
 from datasets import Dataset
-from datasets.exceptions import DatasetNotFoundError
```

Comment:
No longer exists. The package now throws `FileNotFoundError` as demonstrated below